trip_app/trips/tests/conftest.py

```diff
@@ -17,10 +19,12 @@ def trips(request):
 def trip_data():
     """ Create a trip model without touching database and return its dict
         repr for using as a request payload """
-    trip = TripFactory.build()
-    serializer = TripSerializer(trip)
-    data = serializer.data
-    del data["id"]
-    del data["driver"]
-    del data["passengers"]
+    data = {
+        "dep_time": (timezone.now() + timedelta(days=1)),
+        "start_point": f"{faker.latitude()} {float(faker.longitude())}",
+        "dest_point": f"{faker.latitude()} {float(faker.longitude())}",
+        "price": faker.random_number(),
```

Comment:
If you change price to `decimal` type, do not forget to change code here too.

Reply:
Changed here as well. Thanks.